internal/server/http/auth/handlers.go

```diff
@@ -74,32 +74,45 @@ func (h *Handlers) CheckPasswordCode(w http.ResponseWriter, r *http.Request) {
 	h.render(w, tk)
 }
 
-func (h *Handlers) ChangePassword(w http.ResponseWriter, r *http.Request) {
-	var req entity.UserNewPassword
-	if err := json.NewDecoder(r.Body).Decode(&req); err != nil {
-		h.writeErrorResponse(http.StatusBadRequest, "could not parse request", w)
+// GetToken by basic auth.
+func (h *Handlers) GetToken(w http.ResponseWriter, r *http.Request) {
+	var newUser entity.NewUser
```

Comment:
as I understood, you moved login and logout to auth handlers, and change password to user. It is ok. 
However, we should think, how we would divide our db into several smaller dbs. If it is going to touch user, we should move it again

Reply:
it is for further research))

Reply:
any suggestion ?

Reply:
I thought about dividing user from his password. Auth service will handle it

Reply:
ye, password and password hash

Reply:
i need to divide http layer, it's too mixed with business logic now

Reply:
ok, good luck then.

Reply:
any help ? :D

Reply:
for now i will wait for other branch. then we could make changes separetly. Or you wouldn't touch all code?

Reply:
if so, I can start

Reply:
i can do everything that depends on user and leave auth part on you, or any idea ?

Reply:
it depends on what part you need, if not auth, I will do something with it next week